src/Microsoft.Azure.SignalR.Common/Microsoft.Azure.SignalR.Common.csproj

```diff
@@ -11,7 +11,13 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Azure.SignalR.Protocols\Microsoft.Azure.SignalR.Protocols.csproj" />
+    <ProjectReference Include="..\Microsoft.Azure.SignalR.Serverless.Protocols\Microsoft.Azure.SignalR.Serverless.Protocols.csproj" />
```

Comment:
where do we need this serverless protocol dependency?

Reply:
removed in #1684